demos/common/cpp/utils/include/utils/slog.hpp

```diff
@@ -79,10 +81,20 @@ class LogStream {
         (*_log_stream) << std::boolalpha;
         return *this;
     }
+
+    // Specializing for std::vector and std::list
+    template<template<class, class> class Container, class T>
+    LogStream& operator<< (const Container<T, std::allocator<T>>& container) {
```

Comment:
Implementation of `presenter.reportMeans()` was changed to allow print `[ INFO ]` on each new string in python demos also.  And in case of string we can't insert `slog::endl` into it for cpp demos.